components/main-menu.tsx

```
@@ -1,13 +1,14 @@
 
-import React, { useState, useCallback, useMemo } from 'react';
+import React, { useState, useCallback, useMemo, useEffect } from 'react';
 import { motion } from 'framer-motion';
 import { Button } from '@/components/ui/button';
 import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
 import { CountrySelection } from '@/components/country-selection';
 import { GameMode, CountrySettings } from '@/lib/types';
+import { useGameStore } from '@/lib/storage/store';
 import { logger } from '@/lib/logger';
-import { MapPin, Play, Trophy, Settings, Info, Infinity, Activity, Globe, Heart } from 'lucide-react';
+import { MapPin, Play, Trophy, Settings, Info, Infinity, Activity, Globe, Heart, Eye, EyeOff } from 'lucide-react';
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)